fuzzers/binary_only/qemu_launcher/src/instance.rs

```diff
@@ -186,6 +186,7 @@ impl<M: Monitor> Instance<'_, M> {
         let time_observer = TimeObserver::new("time");
 
         let map_feedback = MaxMapFeedback::new(&edges_observer);
+        let map_objective = MaxMapFeedback::new(&edges_observer);
```

Comment:
I think this won't work as you expect.

MaxMapFeedback will use named metadata MapFeedbackMetadata for remember the coverage map
but this name is taken from the observer name.

This means that you made two feedbacks `map_feedback` and `map_objective` but you will have only 1 `MapFeedbackMetadata` for the coverage map while you actually want to metadata, one for feedback, one for objective

Reply:
We should just make a global var to count how each object has been generated (not just here, but in all places that used named metadata)

Reply:
Good point, fixed